OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -70,7 +70,7 @@ void ILintMapPass.Run(Action<string> emitError, Action<string> emitWarning, ModD
 				if (modTranslation.HasMessage(key))
 				{
 					if (mapTranslation.HasMessage(key))
-						emitError($"Map level `{language}` translation key `{key}` will not be used.");
+						emitWarning($"Map level `{language}` translation key `{key}` will not be used.");
```

Comment:
This should be reverted. As discussed on Discord the message could be a bit clearer though, maybe something along the lines of `{language} translation key `{key}` cannot be overwritten on a map level.`?